src/ui/common/src/utils/integrations.ts

```diff
@@ -177,7 +177,8 @@ export type Service =
   | 'Airflow'
   | 'Kubernetes'
   | 'SQLite'
-  | 'Lambda';
+  | 'Lambda'
+  | 'Google Sheets';
```

Comment:
this gives me compiling error when trying to compare a nobject with `Server` type to `Google Sheets`. I'm not sure why this didn't happen before but this fix should make sense.